neotron-bmc-pico/src/main.rs

```diff
@@ -468,6 +478,12 @@ mod app {
 					// TODO: Copy byte to software buffer and turn UART RX
 					// interrupt off if buffer is full
 				}
+				Some(Message::SpeakerConfigReset) => {
```

Comment:
Maybe we should also stop the PWM hardware here. Saves having to lock the object in the timer callback, if the only thing the timer callback does is a message post.

Reply:
Yeah, makes sense.